app/src/main/res/layout/exploration_activity.xml

```diff
@@ -1,8 +1,43 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/exploration_fragment_placeholder"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  tools:context=".player.exploration.ExplorationActivity" />
+  tools:context=".player.exploration.ExplorationActivity">
+
+  <com.google.android.material.appbar.AppBarLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.appcompat.widget.Toolbar
+      android:id="@+id/exploration_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/colorPrimary"
+      android:theme="@style/Widget.AppCompat.ActionBar">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+          android:id="@+id/enable_audio_playback_button"
+          android:layout_width="24dp"
+          android:layout_height="24dp"
+          android:minWidth="48dp"
+          android:minHeight="48dp"
+          android:contentDescription="@string/enable_audio_playback_content_description"
+          android:src="@drawable/ic_volume_off_48dp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+      </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.appcompat.widget.Toolbar>
+  </com.google.android.material.appbar.AppBarLayout>
+
+  <FrameLayout
+    android:id="@+id/exploration_fragment_placeholder"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior" />
```

Comment:
'appbar_scrolling_view_behavior` doesn't seem to be present in strings.xml file.

Reply:
Good catch--it's defined in a strings.xml file we pull in from the material library. It should be ctrl + clickable in Android Studio once you sync Gradle & rebuild. I don't quite like how they implemented this mechanism since it's confusing to reference a string from outside the project, but it's correct and works.